sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -1642,6 +1653,9 @@ def get_transceiver_loopback(self):
         host_input_loopback_lane8   = BOOLEAN                          ; host side input loopback enable lane8
         ========================================================================
         """
+        loopback_support = self.get_transceiver_loopback_support()
+        if loopback_support is None:
+            return None
         trans_loopback = dict()
         trans_loopback['media_output_loopback'] = self.get_media_output_loopback()
```

Comment:
please add check for get_media_output_loopback() and other loopback APIs by checking the advertisement in page13, byte128

Reply:
done